bedrock/firefox/templates/firefox/desktop/tips.html

```diff
@@ -36,7 +36,7 @@
   <div id="page-header">
     <div class="container">
       <div id="heading-logo-wrapper">
-        <img src="{{ static('img/firefox/common/firefox-logo.png') }}" alt="Firefox" id="page-header-logo">
+        <img src="{{ static('img/firefox/template/logo-high-res.png') }}" alt="Firefox" id="page-header-logo">
```

Comment:
`-high-res` filenames are usually reserved for high-dpi images served via a helper or mixin. The logo also looks a little egged here. Changing this from `logo-high-res.png` to `logo-large.png` seems a better fit.